plugins/inputs/pgbouncer/README.md

```diff
@@ -4,6 +4,31 @@ This PgBouncer plugin provides metrics for your PgBouncer load balancer.
 
 More information about the meaning of these metrics can be found in the [PgBouncer Documentation](https://pgbouncer.github.io/usage.html)
 
+Deprecated (_NEXT_VERSION_): use the [postgresql_extensible](../postgresql_extensible) input.
```

Comment:
I don't think we would want to deprecate this plugin even if it can be reproduced with the postgresql_extensible plugin, though maybe we will reimplement it in terms of the other other plugin.  For now though let not add this comment.

Reply:
Thanks. Done. Rename header with: "Reproduce behaviour with postgresql_extensible input"